### PR TITLE
Added an small molecules dataset

### PR DESCRIPTION
**Pull request by DaniDelHoyo**

### CODE REVIEW

**Review by JorMaister — APPROVED** [review]

File: pwem/tests/__init__.py [hunk 1]
```diff
@@ -283,3 +283,11 @@ def defineDatasets():
                 'allMics': 'micrographs/*.mrc',
                 'autoPickingCoordinates': 'autoPickingCoordinates'
             })
+
```
Comment (pconesa) [inline pwem/tests/__init__.py:286]: Datasets can be define inside plugins. This is not the right place for this one. ;-)

Comment (DaniDelHoyo) [reply pwem/tests/__init__.py:286]: Hi Pablo, I tried defining it inside the scipion-chem plugin but seems like it didn't work.
I guess the datasets definition goes over some plugins but maybe not over the scipion-chem, that's why I added it here.
Should we change it otherway?

**Comment by pconesa** [comment]
Have you tried this:

https://github.com/scipion-em/scipion-em-continuousflex/blob/master/continuousflex/__init__.py#L129

**Comment by DaniDelHoyo** [comment]
Okay, defining it in the plugin/__init__() instead of in the plugin/tests/__init__() did work. Thank you

**Pull request merged**